rhel8/profiles/ism_o.profile

```diff
@@ -137,22 +111,27 @@ selections:
   - openssl_use_strong_entropy
 
   ## Network design and configuration
-  ## Identifiers 1311 
+  ## Identifiers 1055 / 1311 
+  - network_nmcli_permissions
   - service_snmpd_disabled
   - snmpd_use_newer_protocol
 
   ## Wireless networks
-  ## Identifiers 1315 / 1319
+  ## Identifiers 1315
   - wireless_disable_interfaces
-  - network_ipv6_static_address
 
-  ## ASD Approved Cryptopgraphic Algorithims
-  ## Identifiers 1446
-  - enable_dracut_fips_module
+  ## ASD Approved Cryptographic Algorithms
+  ## Identifiers 0471 / 0472 / 0473 / 0474 / 0475 / 0476 / 0477 / 
+  ## 0479 / 0480 / 0481 / 0489 / 0497 / 0994 / 0998 / 1001 /  1139 / 
+  ## 1372 / 1373 / 1374 / 1375
   - enable_fips_mode
   - var_system_crypto_policy=fips
   - configure_crypto_policy
+  - sshd_use_approved_ciphers
```

Comment:
I think this is not needed because ciphers are configured by crypto-policies. You already use rule configure_crypto_policy so this should be covered. We even do not have rhel8 remediation for this rule.

Reply:
Thanks @vojtapolasek 

The intent here was to align more closely with the [ASD Approved Cryptographic Algorithms](): https://www.cyber.gov.au/acsc/view-all-content/guidance/asd-approved-cryptographic-algorithms

You're right though - [FIPS](https://csrc.nist.gov/csrc/media/publications/fips/140/2/final/documents/fips1402annexa.pdf) only permits these, so I think we're good here using the FIPS system-wide crypto policy.

I'll make these changes and rebase.